### PR TITLE
Fixes #85 Added profile links and indented

### LINKED ISSUE #85: Add missing links and indent all codes in index.html
There are lot of missing profile links. I'll try to add those.
And indent all in index.html
# I am working on this

### PR DESCRIPTION
**Pull request by supunTE**
https://github.com/fossasia/2015.fossasia.org/issues/85
  https://stedirisuriya.github.io/2015.fossasia.org/

### CODE REVIEW

**Review by vaibhavsingh97 — COMMENTED** [review]
Cab you please tell me what profile link you added. You can point the line number

**Comment by supunTE** [comment]
@vaibhavsingh97 
1058, 1061 -2 links
1114, 1117 -2 links
1397, 1400 -2 links
1788, 1791, 1794 -3 links
1857 -link
1936 -link

total 11 links

File: index.html [hunk 1]
```diff
@@ -287,2170 +412,2990 @@ <h1>Proudly sponsored by</h1>
...
+											<a target="_self" href="https://www.facebook.com/rasmusskjoldan">
+												<i class="icon social_facebook"></i>
+											</a>
+											<a href="https://conference.magnolia-cms.com/2016/san-francisco/speakers/rasmus-skjoldan.html">
+												<i class="icon  icon_link_alt" aria-hidden="true"></i>
+											</a>
+											<a href="https://www.slideshare.net/rasmusskjoldan">
+												<i class="icon  icon_link_alt" aria-hidden="true"></i>
+											</a>
+											<a href="https://www.slideshare.net/rasmusskjoldan">
```
Comment (vaibhavsingh97) [inline index.html]: Profile links are reapeated. PLease remove one

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

**Review by vaibhavsingh97 — APPROVED** [review]

**Review by arjunpandey2003 — APPROVED** [review]
LGTM!

**Review by NaviRocker — CHANGES_REQUESTED** [review]
Please Remove Unwanted Lines from the Code

**Pull request merged**